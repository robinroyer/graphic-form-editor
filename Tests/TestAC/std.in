C c01 10 10 5
EXIT