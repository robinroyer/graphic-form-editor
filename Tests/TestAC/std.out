OK
#New object: c01